product_sequence/readme/USAGE.md

```diff
@@ -0,0 +1,14 @@
+To specify a different sequence for a product category proceed as
+follows:
+
+1.  Go to the a Product Category form view. (**note:** you will need to
+    install Inventory app to be able to access to the form view,
+    *Inventory \> Configuration \> Products \> Products Categories*; or
+    create a menuitem manually).
+2.  Fill the *Prefix for Product Internal Reference* as desired.
+3.  Under the settings (Settings -\> General Settings -\> Product
+    Sequences), you can specify whether the prefix of the parent
+    category should be used if no prefix has been specified for the
+    category.
+
+[![Try me on Runbot](https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas)](https://runbot.odoo-community.org/runbot/135/18.0)
```

Comment:
You can remove this. It's already being added by the README generator as a badge at the top of the README.rst.